vcpkg.json

```diff
@@ -1,12 +1,28 @@
 {
-    "name": "openloco",
-    "version-string": "22.02",
-    "dependencies": [
-        "benchmark",
-        "breakpad",
-        "gtest",
-        "libpng",
-        "openal-soft",
-        "sdl2"
-    ]
+  "name": "openloco",
+  "version-string": "22.02",
+  "dependencies": [
+    "benchmark",
+    {
+      "name": "breakpad",
+      "platform": "windows"
+    },
+    "gtest",
+    "libpng",
+    "openal-soft",
+    {
+      "name": "sdl2",
+      "default-features": false,
+      "features": [
+        "x11"
```

Comment:
Very disappointingly I had to specifically disable the Wayland feature from sdl2, otherwise it'll pull in dbus (and a lot of others), and dbus cannot be statically linked. Honestly, I'm not sure this will ever be solvable, and is probably the biggest downside to static linking on Linux.

Reply:
Well never mind, I managed to add wayland back in. Shockingly, the problem with dbus wasn't the wayland feature of sdl2 in vcpkg but the.... dbus feature :facepalm:

I've even confirmed the game works ok with the resulting binary when run with `SDL_VIDEODRIVER=wayland`